drafts/use-cases/3.obtaining-events.md

```diff
@@ -0,0 +1,110 @@
+## Entry point
```

Comment:
The title should be changed and likely the document's name too. What you are describing here is retrieving members of a collection so maybe you wanna use that instead.